express-basic/server.js

```diff
@@ -11,20 +11,16 @@ const app = Express();
 
 app.use("/color/:name", function (req, res) {
   let name = req.params.name;
-  console.log("NAME", name);
+  console.log("NAME", name.toLocaleLowerCase());
 
   let code = "#cccccc";
-
-  // if('red' === name) {
-  //   code = '#ff0000'
-  // } else if('green' === name) {
-  //   code = '#00ff00'
-  // } else if('blue' === name) {
-  //   code = '#0000ff'
-  // }
-
-  //Using switch statement.
   switch (name) {
+    case "AliceBlue":
```

Comment:
how will you handle all the other colors?

Reply:
I'll figure it out and push the update, I'll ping you on slack.